src/ankita/assignment/string/ReplaceSecondLastOccurance.java

```diff
@@ -0,0 +1,26 @@
+/*Assignment - 47 : 27th Dec'2020
+
+Replace second last occurance of "Hi" with your name.
+String msg = "Hi globant Hello Hi Hi globant Hi Hello Hello globant Hi Techno Credits Techno";
+output : Hi globant Hello Hi Hi globant Maulik Hello Hello globant Hi Techno Credits Techno
+*/
+package ankita.assignment.string;
+
+public class ReplaceSecondLastOccurance {
+	void replaceScndOccurance(String msg) {
+		System.out.println("Input : " + msg);
+
+		int index1 = msg.lastIndexOf("Hi");
```

Comment:
Hi @AnkitaDOct2020
here you can use replace method as well but you need to this assignment using Collection's method